src/Libraries/DSCPython/CPythonEvaluator.cs

```diff
@@ -101,6 +106,17 @@ public static class CPythonEvaluator
             }
         }
 
+        /// <summary>
+        /// Registers custom encoders and decoders with the Python.NET runtime
+        /// </summary>
+        private static void InitializeEncoders()
+        {
+            var encoders = new IPyObjectEncoder[] { new BigIntegerEncoder() };
```

Comment:
@aparajit-pratap these are the encoders I mentioned before.

Reply:
Was the biginteger encoder/decoder added by you? I may have missed it in your PR's in Python.NET repo.

Reply:
`BigIntegerEncoder` is part of this submission. The API to add them already existed. One thing I did add in Python.NET is the function `PyLong.ToBigInteger`, which is used by the decoder.